src/main.css

```diff
@@ -284,13 +284,14 @@ main {
 
 /* Text-level semantics */
 
-a, .\<a\> {
+a:not(.\<button\>), .\<a\> {
 	color: var(--link-fg, var(--accent));
 	border-radius: var(--border-radius);
 	outline-offset: 1px;
 	background: none;
```

Comment:
Another specificity issue here, where this causes `a.chip` to render without backgrounds or borders.

I think it might be best for me to revert most of the changes and address `a.<button>` and `button.<a>` as separate edge cases (wouldn't be the entire button/link styles there, just the ones that need to be reset), instead of trying to shoe-horn it all in.

Reply:
Fixed this in following commit.